### PR TITLE
Add Docker image

### PR DESCRIPTION
**Pull request by ponelat**

### CODE REVIEW

**Comment by ponelat** [comment]
@jasmingacic I can't recall how to tweak your pull-request, so I created a separate one. 
Let me know if this suits your needs, and you're more than welcome to copy the files over to your own PR and we can merge that.

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+			  desc = descArr[2]; \
+				color = "33m"; \
+			} \
+			gsub(/^[ \t]+/,"",dep); \
+			gsub(/^[ \t]+/,"",desc); \
+			printf "\033[%s%-30s\033[0m%s\n",color,dep,desc \
+		}'
+
+docker-build: ## Build docker image of ponelat/overlays-cli
+	$(RUN) "docker build -f Dockerfile -t ponelat/overlays-cli ."
```
Comment (jasmingacic) [inline Makefile]: Ideally you would run
```
docker buildx build --platform ${PLATFORM} 
```
to build multi arch docker images like for example https://hub.docker.com/r/kubeshop/overlay-cli/tags

To achieve that you can do something similar to:
```
PLATFORM?=linux/amd64,linux/arm64
docker-build:
   docker buildx create --name overlay-cli
   docker buildx use overlay-cli
   docker buildx build --platform ${PLATFORM} -t kubeshop/overlay-cli:latest --push .
```

Also for this you'd have to enable GH action so the image is build every time there is a release tag or similar

Comment (mbana) [reply Makefile]: > Ideally you would run
> 
> ```
> docker buildx build --platform ${PLATFORM} 
> ```
> 
> to build multi arch docker images like for example https://hub.docker.com/r/kubeshop/overlay-cli/tags
> 
> To achieve that you can do something similar to:
> 
> ```
> PLATFORM?=linux/amd64,linux/arm64
> docker-build:
>    docker buildx create --name overlay-cli
>    docker buildx use overlay-cli
>    docker buildx build --platform ${PLATFORM} -t kubeshop/overlay-cli:latest --push .
> ```
> 
> Also for this you'd have to enable GH action so the image is build every time there is a release tag or similar

@ponelat, I'd be interested in this getting merged along with the comment suggested above.

**Comment by ponelat** [comment]
Hi @jasmingacic  and @mbana 
Updated to support multi-arch docker images, so `0.8.1` should have linux/arm64 and linux/amd64. Learned a few new things regarding docker there!

No GH Action support yet, but should be straight forward, as `make docker-release` should do most things.

**Pull request merged**